### DISCUSSION TITLE
Recapture Output Directory

### DISCUSSION THREAD

**Discussion by jarommadsen** [CrankshaftForSteam/crankshaft#23] [General]
Is there any way to change the output directory for recapture? I don’t have much room on my internal drive (64gb Steam Deck here) and would rather use my sd card.

**Comment by Olyve** [comment]
@coolavery wrote that plugin so they might have a better idea! Its possible that it might need to be updated to support that.

**Comment by jarommadsen** [comment]
I found the `-o` option and added it to the index file but I get an error. I'm guessing it doesn't have access to the SD Card. Is that something that can be provided by crankshaft?

**Reply by Olyve** [reply]
hmm, I'm not sure cause I haven't used it. I do know that the path to the SD card is a little odd in the file system. Have you checked the path for your system on the desktop side?

**Reply by coolavery** [reply]
Recapture should be running outside of the Flatpak sandbox so it should have full access to your system. You should be able to use the `-o` option like you said.

Can you try manually running Recapture from a terminal/SSH, change the `-o` option, and see if it works? Sorry, the command is really long, but you can change the output path at the end:

```bash
GST_VAAPI_ALL_DRIVERS=1 GST_PLUGIN_PATH=/home/deck/.var/app/space.crankshaft.Crankshaft/data/crankshaft/plugins/recapture/dist/deps/plugins LD_LIBRARY_PATH=/home/deck/.var/app/space.crankshaft.Crankshaft/data/crankshaft/plugins/recapture/dist/deps/lib /home/deck/.var/app/space.crankshaft.Crankshaft/data/crankshaft/plugins/recapture/dist/deps/recapture record -o=/home/deck/somewhere
```

The command currently can't be stopped with ctrl+c, you'll have to run `pkill recapture` in another terminal to make it stop recording.

**Comment by xftsfyi** [comment]
Hi, if anyone is still having this problem, I've found an easy solution.

Create a folder on your SD card (name it whatever you want)
Create a link to that folder via the right click menu
Copy that folder link over to ~/Videos
Rename the link as "recapture" (delete the current recapture folder if you haven't already)

Your screen recordings should now save to your SD card.
